backend/LexBoxApi/GraphQL/OrgMutations.cs

```diff
@@ -247,6 +249,30 @@ public async Task<IQueryable<Organization>> ChangeOrgMemberRole(
         return dbContext.Orgs.Where(o => o.Id == orgId);
     }
 
+    [Error<NotFoundException>]
+    [Error<LastMemberCantLeaveException>]
+    [UseMutationConvention]
+    [RefreshJwt]
+    public async Task<Organization> LeaveOrg(
```

Comment:
This is largely just a copy/paste of the `LeaveProject` mutation. Having a seperate mutation for this makes both permission and client-cache handling simpler and easier to understand.